File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame10Luglio2017/Azienda.java
public class Azienda extends Cliente {

    private String ragioneSociale;
    private int partitaIva, numeroDipendenti;
    private String IBAN;
    
    public Azienda(int codice, String indirizzo, String ragioneSociale, int partitaIva, int numeroDipendenti, String IBAN) {
        super(codice, indirizzo);
        this.ragioneSociale = ragioneSociale;
        this.partitaIva = partitaIva;
        this.numeroDipendenti = numeroDipendenti;
        this.IBAN = IBAN;
    }

    public String toString() {
        return "Azienda" + "\t-\t-\t" + ragioneSociale + "\t" + partitaIva + "\t" + numeroDipendenti + "\t" + IBAN + "\t" + indirizzo;
    }
    
}

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame29Gennaio2016/Python/Azienda.py
from Cliente import Cliente

class Azienda(Cliente):

    ragione_sociale: str
    fatturato: int

    def __init__(self, codice, indirizzo, giorno, mese, anno, ragione_sociale, fatturato):
        super().__init__(codice, indirizzo, giorno, mese, anno)
        self.ragione_sociale = ragione_sociale
        self.fatturato = fatturato